src/sentry/incidents/endpoints/serializers.py

```diff
@@ -165,6 +165,7 @@ def validate(self, attrs):
                     {"sentry_app": "SentryApp must be provided for sentry_app"}
                 )
         attrs["use_async_lookup"] = self.context.get("use_async_lookup")
+        attrs["input_channel_id"] = self.context.get("input_channel_id")
```

Comment:
We should probably validate that this channel id is valid rather than just accepting it. Should be a pretty quick api call to do so

Reply:
My thoughts here:
 - Generally a bad idea to accept unvalidated ids. Security implications etc. Probably not a huge risk here because we'll be scoped to a particular integration.
 - We can provide better feedback to them if we're unable to access the channel - permission issues etc. We could also get the channels actual name and make sure it isn't different to what was passed.
  - Related: Anyone using this is likely to be a large customer, since it tends to correlate to having a large slack installation. It's better if we can provide them with a better experience in general, otherwise we might be diagnosing why certain channels don't work, and wondering how they added something invalid.

Reply:
@ceorourke @wedamija rethinking it and probs makes sense to hit the `conversations.info` api (for both). i think maybe at the time we did this for issue alerts we weren't fully using the `conversations` api.

Reply:
I'll make a ticket for this